macros/TEST_SPEED.cfg

```
@@ -63,49 +63,51 @@ gcode:
     G0 X{x_min} Y{y_min} Z{bound + 10} F{speed*60}
 
     # Set new limits
-    SET_VELOCITY_LIMIT VELOCITY={speed} ACCEL={accel} ACCEL_TO_DECEL={accel / 2}
+    SET_VELOCITY_LIMIT VELOCITY={speed} ACCEL={accel} #MIN_CRUISE_RATIO = {0.5} #ACCEL_TO_DECEL={accel / 2}
```

```suggestion
    SET_VELOCITY_LIMIT VELOCITY={speed} ACCEL={accel} MINIMUM_CRUISE_RATIO=0
```
We want to disable the minimum cruise ratio, right? To ensure we actually move at the speed and accel that this macro wants.